spec/rails_helper.rb

```diff
@@ -35,7 +35,7 @@
 end
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = "#{Rails.root}/spec/fixtures"
+  config.fixture_paths = ['spec/fixtures']
```

Comment:
Nice update switching from `fixture_path` to `fixture_paths`. Rails now expects an array, which also makes it easier to support multiple fixture directories in the future. 👍